tests/test_constants.py

```diff
@@ -21,39 +37,419 @@ def test_constants(self):
         """
         self.maxDiff = None
 
-        self.assertEqual(constants.CMD_CURRENT_APP, "CURRENT_APP=$(dumpsys window windows | grep mCurrentFocus) && CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && echo $CURRENT_APP")
+        # CMD_CURRENT_APP
+        self.assertEqual(constants.CMD_CURRENT_APP, "CURRENT_APP=$(dumpsys window windows | grep -E 'mCurrentFocus|mFocusedApp') && CURRENT_APP=${CURRENT_APP#*ActivityRecord{* * } CURRENT_APP=${CURRENT_APP#*{* * } && CURRENT_APP=${CURRENT_APP%%/*} && CURRENT_APP=${CURRENT_APP%\}*} && echo $CURRENT_APP")
```

Comment:
I would say the direct reference to the constant is suffice, do we really need to add the comment as well?

Reply:
The comment helps with git diffs.  https://github.com/JeffLIrion/python-androidtv/commit/abe39dec6e7a889f96abfa35295f6fd6507dfa4f